e2e/test/netstandard20/ProvisioningServiceClientE2ETests.cs

```diff
@@ -19,8 +20,14 @@ namespace Microsoft.Azure.Devices.E2ETests
     [TestCategory("Provisioning-E2E")]
     public class ProvisioningServiceClientE2ETests : IDisposable
     {
+        public enum AttestationType
```

Comment:
We should add None as a type here?

Reply:
None isn't a valid user scenario that is worth testing, so I left it out. The service should reject any created enrollment with attestation type "None" correct?